src/pages/List.js

```diff
@@ -105,16 +105,26 @@ export default function List({ token }) {
 
   function deleteItem(id) {
     Swal.fire({
-      title: 'Are you sure?',
-      text: "You won't be able to revert this!",
+      title: 'Are you sure you want to delete this item from your list?',
+      text: "Once it's gone, it's gone!",
       icon: 'warning',
+      iconColor: '#F5AB00',
       showCancelButton: true,
-      confirmButtonColor: '#3085d6',
-      cancelButtonColor: '#d33',
-      confirmButtonText: 'Yes, delete it!',
+      reverseButtons: true,
+      confirmButtonColor: '#073B4C',
+      cancelButtonColor: '#EB1E4E',
+      cancelButtonText: 'Do not delete this item',
+      confirmButtonText: 'Yes, delete this item!',
```

Comment:
This looks great! A lot like I did with the update to the `deleteItem` sweet alert I played around with. We can decide as a group what we like better overall, but I do want to mention one thing: I think it makes more sense to have the `confirmationButtonColor` be a destructive red color, but I want everyone else's input on that too. What do you think @apdsrocha and @zeeatwork ??

![delete-swa-after](https://user-images.githubusercontent.com/47455758/119994462-3cbbc080-bf92-11eb-9f49-44631909f765.jpg)

Blue is "safe" to me and red is "you done did it!" 

Also, do you like passing in the item name to the sweet alert?

Reply:
Definitely like passing the item name into the Sweet Alert. Let's add that. 

And yes, the confirmation color is kind of throwing me for a loop. Ultimately, I do find the red color as a representation of "stop whatever path we are currently traveling down right now". 

Also the placement of the confirmation vs cancel action buttons, I've noticed that the confirmation of the action button tends to be on the right, and cancel the action button tends to be on the left.

Reply:
@jamesncox @jssckbl This is the kind of thing that would largely benefit from having a UX-er do some user research and see how people understand these buttons 😆 What I would do right now is probably leave both buttons with the blue background and rely on the labels to guide the user. There's this cool article on primary and secondary buttons (right [here](https://uxplanet.org/primary-secondary-action-buttons-c16df9b36150)) that has a lot of views on this, but I'd go with "yes delete" on the right, "cancel" on the left, both in the same color we've been using on all buttons. But again, I don't think there's a right/wrong call here, so if anyone feels strongly about it, I say go ahead with whatever feels right :D

Reply:
@apdsrocha that's such an interesting article and great read! I agree about having the "yes delete" on the right and "cancel" on the left. Seems like most sites follow that same flow. 

As far as the button color itself goes, I'm okay with the same color. I don't have particularly strong feelings about the colors that much. But the position, now that you bring it up, I have strong feelings about that!

Reply:
The article you linked was really helpful, @apdsrocha. 

I see why I have the urge to have the cancel button on the left, and confirm on the right, since I spend most of my time on Apple products, and I also use Google products. 

I'm going to align the buttons with cancel on the left and yes complete the action on the right. And I'll try out the blues, and maybe a gray for the color for the complete the action button on the right, and just push what looks good to me at the time, knowing we can always still change the colors 😸